apps/coscrad-frontend/src/emotion-components/index.ts

```diff
@@ -0,0 +1,9 @@
+import styled from '@emotion/styled';
+import { Card } from '@mui/material';
+import { primaryColor as primary } from './colors';
+
+export const PrimaryCard = styled(Card)`
```

Comment:
This is very interesting. You are wrapping in styling. But this also could be where we implement any other tweaks to the underlying MUI Card to define a generic card like `ResourceDetailFullViewCard`.

Then from outside here, every `detail full view` component is created by adapting the View Model interface to the `ResourceDetailFullViewCard`. This would give us the consistency we are looking for in presentation and styling in one shebang- a single source of truth.